src/components/streams/StreamPlayer.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react'
+
+import DrawViewContainer from '../../containers/draw/DrawViewContainer'
+
+import { setColor } from '../../actions/colors'
+import { setThickness } from '../../actions/thickness'
+
+
+export default React.createClass({
+  getInitialState () {
+    return {
+      currentLine: null,
+      currentColor: 'red',
```

Comment:
This is some leftover stuff from testing, but I'll just let it get swept up when I redo some things.